polyfill/intersectionobserver-polyfill.js

```diff
@@ -58,6 +58,12 @@
         ancestor = ancestor.parentNode;
       }
 
+      this._mutationObserver.observe(target, {
+        attributes: true,
+        childList: true,
+        characterData: true,
```

Comment:
Do you need to observe `characterData` changes to track the target element's visibility? Can you use `attributeFilter` to observe only changes in the target's dimension attributes?